Application randomly crash on KiwixMainActivity

**Describe the bug**
Application randomly crash on KiwixMainActivity with error

**Logs**


**Expected behavior**
It should not crash

**Steps to reproduce the behavior:**
1. It's randomly crash when i install latest develop branch apk over old branch apk in emulator.

**Environment**
- Version of Kiwix Android : Latest develop branch
- Device : emulator
- OS version : Android 21 

**Logs**
```
Process: org.kiwix.kiwixmobile, PID: 22131
                                                                                                    java.lang.RuntimeException: Unable to start activity ComponentInfo{org.kiwix.kiwixmobile/org.kiwix.kiwixmobile.main.KiwixMainActivity}: kotlin.UninitializedPropertyAccessException: lateinit property activityKiwixMainBinding has not been initialized
                                                                                                    	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3780)
                                                                                                    	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3947)
                                                                                                    	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:106)
                                                                                                    	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
                                                                                                    	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
                                                                                                    	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2325)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:106)
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:210)
                                                                                                    	at android.os.Looper.loop(Looper.java:299)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8280)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:576)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1073)
                                                                                                    Caused by: kotlin.UninitializedPropertyAccessException: lateinit property activityKiwixMainBinding has not been initialized
                                                                                                    	at org.kiwix.kiwixmobile.main.KiwixMainActivity.onCreate(KiwixMainActivity.kt:113)
                                                                                                    	at android.app.Activity.performCreate(Activity.java:8163)
                                                                                                    	at android.app.Activity.performCreate(Activity.java:8130)
                                                                                                    	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1329)
                                                                                                    	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3753)
                                                                                                    	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3947) 
                                                                                                    	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:106) 
                                                                                                    	at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135) 
                                                                                                    	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95) 
                                                                                                    	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2325) 
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:106) 
                                                                                                    	at android.os.Looper.loopOnce(Looper.java:210) 
                                                                                                    	at android.os.Looper.loop(Looper.java:299) 
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:8280) 
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method) 
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:576) 
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1073) 
```

--- a/app/src/main/java/org/kiwix/kiwixmobile/main/KiwixMainActivity.kt
+++ b/app/src/main/java/org/kiwix/kiwixmobile/main/KiwixMainActivity.kt
@@ -110,19 +110,21 @@ class KiwixMainActivity : CoreMainActivity() {
 
   override fun onConfigurationChanged(newConfig: Configuration) {
     super.onConfigurationChanged(newConfig)
-    activityKiwixMainBinding.bottomNavView.menu.apply {
-      findItem(R.id.readerFragment)?.title = resources.getString(R.string.reader)
-      findItem(R.id.libraryFragment)?.title = resources.getString(R.string.library)
-      findItem(R.id.downloadsFragment)?.title = resources.getString(R.string.download)
-    }
-    activityKiwixMainBinding.drawerNavView.menu.apply {
-      findItem(R.id.menu_bookmarks_list)?.title = resources.getString(R.string.bookmarks)
-      findItem(R.id.menu_history)?.title = resources.getString(R.string.history)
-      findItem(R.id.menu_notes)?.title = resources.getString(R.string.pref_notes)
-      findItem(R.id.menu_host_books)?.title = resources.getString(R.string.menu_wifi_hotspot)
-      findItem(R.id.menu_settings)?.title = resources.getString(R.string.menu_settings)
-      findItem(R.id.menu_help)?.title = resources.getString(R.string.menu_help)
-      findItem(R.id.menu_support_kiwix)?.title = resources.getString(R.string.menu_support_kiwix)
+    if (::activityKiwixMainBinding.isInitialized) {
+      activityKiwixMainBinding.bottomNavView.menu.apply {
+        findItem(R.id.readerFragment)?.title = resources.getString(R.string.reader)
+        findItem(R.id.libraryFragment)?.title = resources.getString(R.string.library)
+        findItem(R.id.downloadsFragment)?.title = resources.getString(R.string.download)
+      }
+      activityKiwixMainBinding.drawerNavView.menu.apply {
+        findItem(R.id.menu_bookmarks_list)?.title = resources.getString(R.string.bookmarks)
+        findItem(R.id.menu_history)?.title = resources.getString(R.string.history)
+        findItem(R.id.menu_notes)?.title = resources.getString(R.string.pref_notes)
+        findItem(R.id.menu_host_books)?.title = resources.getString(R.string.menu_wifi_hotspot)
+        findItem(R.id.menu_settings)?.title = resources.getString(R.string.menu_settings)
+        findItem(R.id.menu_help)?.title = resources.getString(R.string.menu_help)
+        findItem(R.id.menu_support_kiwix)?.title = resources.getString(R.string.menu_support_kiwix)
+      }
     }
   }
 

